python/GafferSceneUITest/SceneGadgetTest.py

```diff
@@ -631,6 +631,28 @@ def testResizeWindow( self ) :
 			gw._qtWidget().setFixedWidth( 200 + ( i % 2 ) * 200 )
 			self.waitForIdle( 100 )
 
+	def testSetRenderer( self ) :
+
+		s = Gaffer.ScriptNode()
+		s["s"] = GafferScene.Sphere()
+
+		sg = GafferSceneUI.SceneGadget()
+		sg.setMinimumExpansionDepth( 1 )
+		sg.setScene( s["s"]["out"] )
+
+		with GafferUI.Window() as w :
+			gw = GafferUI.GadgetWidget( sg )
+
+		w.setVisible( True )
+		self.waitForIdle( 1000 )
+
+		for i in range( 0, 5 ) :
+			print( i )
```

Comment:
Is this print intended or a holdover from reproducing the issue?

Reply:
Oops. Totally not intended - removed in https://github.com/GafferHQ/gaffer/pull/5120/commits/9bfece0c88cbae88cd7b8e8093eb1e5d689dc164.